api/java/editor/api-editor/src/main/java/ca/bc/gov/tno/areas/admin/models/ActionModel.java

```diff
@@ -145,14 +145,14 @@ public void setValueLabel(String valueLabel) {
   /**
    * @return boolean the enabled
    */
-  public boolean isEnabled() {
+  public boolean getIsEnabled() {
```

Comment:
Renamed all java objects to correctly handle the name change from `enable` to `isEnabled`.  This was done to enable the Java DAL to continue working even though the Entity Framework DAL is now managing the database migrations.